src/test/java/hudson/plugins/batch_task/RestartTest.java

```diff
@@ -5,13 +5,15 @@
 import jenkins.model.Jenkins;
 import org.junit.Rule;
 import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
 import org.jvnet.hudson.test.JenkinsRule;
 import static org.junit.Assert.*;
 
 
 /**
  * @author Kohsuke Kawaguchi
  */
+@Issue("2917")
```

Comment:
See `@Issue` javadoc. The ID to use must be the complete ID: JENKINS-2917 in this case.